src/inmanta/deploy/persistence.py

```diff
@@ -331,3 +347,15 @@ async def dryrun_update(self, env: UUID, dryrun_result: DryrunResult) -> None:
             messages=dryrun_result.messages,
             status=const.ResourceState.dry,
         )
+
+    async def set_parameters(self, fact_result: FactResult) -> None:
+        await self.client.set_parameters(tid=self.environment, parameters=fact_result.parameters)
+        await self.client.resource_action_update(
+            tid=self.environment,
+            resource_ids=[fact_result.resource_id],
+            action_id=fact_result.action_id,
+            action=const.ResourceAction.getfact,
+            started=fact_result.started,
+            finished=fact_result.finished,
+            messages=fact_result.messages,
+        )
```

Comment:
We could choose to write this to the DB directly, but not required

Reply:
I chose to leave it like this to be consistent with the `dryrun_update` I figured we would do this on the next step. But if you want, we could do this (otherwise I'll create a ticket for it)